### PR TITLE
[innogysmarthome] Fix isr2stop

### PR DESCRIPTION
**Pull request by mmans**
[innogysmarthome] Optimizes the control of Shutters (ISR2) by using the correct api-calls. Adding the possibility to use the "STOP" command when the shutters are moving.

In the current release the binding uses the "SetState" command to move the shutters (When user clicks on arrow buttons of the default sitemap-widget). So in fact it sets the position variable to 0% or 100%. The stop button was not working. In this PR I use the same API-call as Innogy uses in their app to start moving the shutters (RampStart). I also added support for the STOP-button by using the "RampStop" API-call. This was a missing feature.

### CODE REVIEW

File: bundles/org.openhab.binding.innogysmarthome/src/main/java/org/openhab/binding/innogysmarthome/internal/client/entity/action/ShutterAction.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ */
+public class ShutterAction extends Action {
+
+    public enum ShutterActions {
+        UP,
+        DOWN,
+        STOP
+    }
+
+    private final String TYPE_STOP_RAMP = "StopRamp";
```
Comment (J-N-K) [inline bundles/org.openhab.binding.innogysmarthome/src/main/java/org/openhab/binding/innogysmarthome/internal/client/entity/action/ShutterAction.java:28]: why are these not `static`?

File: bundles/org.openhab.binding.innogysmarthome/src/main/java/org/openhab/binding/innogysmarthome/internal/handler/InnogyBridgeHandler.java [hunk 2]
```diff
@@ -110,7 +111,8 @@
     private @Nullable DeviceStructureManager deviceStructMan;
     private @Nullable String bridgeId;
     private @Nullable ScheduledFuture<?> reinitJob;
-    private @NonNullByDefault({}) InnogyBridgeConfiguration bridgeConfiguration;
+    private @NonNullByDefault({})
```
Comment (J-N-K) [inline bundles/org.openhab.binding.innogysmarthome/src/main/java/org/openhab/binding/innogysmarthome/internal/handler/InnogyBridgeHandler.java:114]: Please revert this format change. Single annotations are on the same line.

File: bundles/org.openhab.binding.innogysmarthome/src/main/java/org/openhab/binding/innogysmarthome/internal/client/entity/action/ShutterAction.java (hunk 1 above)
Comment (mmans) [reply bundles/org.openhab.binding.innogysmarthome/src/main/java/org/openhab/binding/innogysmarthome/internal/client/entity/action/ShutterAction.java]: oops :)

File: bundles/org.openhab.binding.innogysmarthome/src/main/java/org/openhab/binding/innogysmarthome/internal/handler/InnogyBridgeHandler.java (hunk 2 above)
Comment (mmans) [reply bundles/org.openhab.binding.innogysmarthome/src/main/java/org/openhab/binding/innogysmarthome/internal/handler/InnogyBridgeHandler.java]: I changed it back.

**Pull request merged**